HW04/maksim_manzhai/spec/student_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'student'
+require 'notification'
+require 'mentor'
+require 'homework'
+
+RSpec.describe Student do
+  subject { described_class.new(name: 'Chris', surname: 'Rock') }
+
+  let(:mentor) { Mentor.new(name: 'Will', surname: 'Smith') }
+  let(:notification) { Notification.new(title: 'HW03', description: 'OOP in Ruby') }
+  let(:homework) { Homework.new(title: 'HW03', description: 'OOP in Ruby', student: subject, mentor: mentor) }
+
+  context 'when student see notification' do
+    it 'sees empty array of notifications' do
+      expect(subject.notifications).to eq([])
+    end
+
+    it 'gets notification' do
+      expect { subject.notifications << notification }.to(change { subject.notifications.size })
```

Comment:
1. you can add the notification in the before block
2. try to use the construction expect {}.to change {}.from().to()

Reply:
in every similar case